src/classes/BDI_DataImportDeleteBTN_CTRL.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>39.0</apiVersion>
```

Comment:
The API version has been set to 37.0. The API version check for new classes was mentioned by Mike in the meeting today.